File: beginner/javascript/2582.js
var fs = require("fs");
var input = fs.readFileSync("stdin", "utf8");
var lines = input.split("\n");

var i, n, n1, n2;

n = parseInt(lines.shift());

for(i = 0; i < n; i++){
    var valores = lines.shift().split(" ");
    n1 = parseInt(valores[0]);
    n2 = parseInt(valores[1]);

    var soma = 0;
    soma = n1 + n2;

    switch (soma){
        case 0: console.log("PROXYCITY"); break;
        case 1: console.log("P.Y.N.G."); break;
        case 2: console.log("DNSUEY!"); break;
        case 3: console.log("SERVERS"); break;
        case 4: console.log("HOST!"); break;
        case 5: console.log("CRIPTONIZE"); break;
        case 6: console.log("OFFLINE DAY"); break;
        case 7: console.log("SALT"); break;
        case 8: console.log("ANSWER!"); break;
        case 9: console.log("RAR?"); break;
        case 10: console.log("WIFI ANTENNAS"); break;
        default: break;
    }
}

File: beginner/java/2582.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int i, n, n1, n2;

        n = scanner.nextInt();

        for(i = 0; i < n; i++){
            n1 = scanner.nextInt();
            n2 = scanner.nextInt();

            int soma = 0;
            soma = n1 + n2;

            switch (soma){
                case 0: System.out.println("PROXYCITY"); break;
                case 1: System.out.println("P.Y.N.G."); break;
                case 2: System.out.println("DNSUEY!"); break;
                case 3: System.out.println("SERVERS"); break;
                case 4: System.out.println("HOST!"); break;
                case 5: System.out.println("CRIPTONIZE"); break;
                case 6: System.out.println("OFFLINE DAY"); break;
                case 7: System.out.println("SALT"); break;
                case 8: System.out.println("ANSWER!"); break;
                case 9: System.out.println("RAR?"); break;
                case 10: System.out.println("WIFI ANTENNAS"); break;
                default: break;
            }
        }
    }
}
